Formula/feedgnuplot.rb

```diff
@@ -16,11 +16,30 @@ class Feedgnuplot < Formula
 
   depends_on "gnuplot"
 
+  uses_from_macos "perl"
+
+  on_linux do
+    resource "List::MoreUtils" do
+      url "https://cpan.metacpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz"
+      sha256 "63b1f7842cd42d9b538d1e34e0330de5ff1559e4c2737342506418276f646527"
+    end
+  end
```

Comment:
Looks like we need the Perl modules' dependencies:
```
==> /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/bin/feedgnuplot --terminal 'dumb 80,20' --exit
Can't locate Exporter/Tiny.pm in @INC (you may need to install the Exporter::Tiny module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/lib/perl5/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/lib/perl5 /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0 /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/5.34.0 /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.34.0) at /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/lib/perl5/List/MoreUtils.pm line 24.
BEGIN failed--compilation aborted at /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/lib/perl5/List/MoreUtils.pm line 24.
Compilation failed in require at /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/bin/feedgnuplot line 12.
BEGIN failed--compilation aborted at /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/libexec/bin/feedgnuplot line 12.
```

`List::MoreUtils` at least requires `Exporter::Tiny` and `List::MoreUtils::XS` based on CPAN https://metacpan.org/pod/List::MoreUtils

```
  resource "List::MoreUtils::XS" do
    url "https://cpan.metacpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-XS-0.430.tar.gz"
    sha256 "e8ce46d57c179eecd8758293e9400ff300aaf20fefe0a9d15b9fe2302b9cb242"
  end
```
```
  resource "Exporter::Tiny" do
    url "https://cpan.metacpan.org/authors/id/T/TO/TOBYINK/Exporter-Tiny-1.002002.tar.gz"
    sha256 "00f0b95716b18157132c6c118ded8ba31392563d19e490433e9a65382e707101"
  end
```